testdata/run_alt_goos_goarch.txt

```diff
@@ -0,0 +1,19 @@
+[!darwin] env GOOS=darwin
+[darwin]  env GOOS=linux
```

Comment:
or just:

    env GOOS=blahblah

?

(and probably worth setting GOARCH too to check that's also overridden).

Reply:
Thanks, yes.